Dockerfile

```diff
@@ -4,34 +4,47 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # Pre-requisites for wkhtmltpdf and our application
 RUN apt-get update -y && apt-get install -y \
+    software-properties-common \
     build-essential \
     xorg \
     libssl-dev \
     libxrender-dev \
-    fontconfig
+    libjpeg-turbo8-dev \
+    fontconfig \
+    xfonts-75dpi \
```

Comment:
New packages?

Reply:
Yes, `libjpeg-turbo8-dev` and `xfonts-75dpi` are required by version **0.12.5** of `wkhtmltopdf`